thehive-templates/TorBlutmagie_1_0/long.html

```diff
@@ -0,0 +1,45 @@
+<div class="panel panel-warning" ng-if="success && content.nodes.length > 0">
+	<div class="panel-heading">
+		Tor nodes - {{artifact.data | fang}}
+	</div>
+	<div class="panel-body">
+		<div ng-repeat="n in content.nodes" class="panel panel-default">
+      <div class="panel-heading">
+        {{n.name}}
+      </div>
+      <div class="panel-body">
+        <dl class="dl-horizontal">
```

Comment:
Please switch the `<dt>` and `<dd>` here:

![grafik](https://user-images.githubusercontent.com/2846609/34094997-eb7f3d9c-e3cf-11e7-9078-be9da4c35348.png)